includes/footer.html

```diff
@@ -60,7 +60,12 @@ <h5 class="white-text fine">coala</h5>
     </div>
     <div class="footer-copyright">
         <div class="container">
-            <span class="left">© 2017 coala</span>
+            <span class="left">© 
+                <script>
+                    document.write(new Date().getFullYear());
+                </script> 
```

Comment:
Trailing Whitespace: Trailing white spaces are unnecessary and can complicate diffs: Remove the ' ' at the end of the line.

*Origin: HTMLLintBear, Section: `html`.*